BT_JV2.js

```diff
@@ -0,0 +1,2 @@
+const nhi = "Tôi năm nay 18 tuổi \nTôi có hoài bão để theo đuổi";
```

Comment:
Need make variable for 18

```suggestion
const age = 18;
const nhi = "Tôi năm nay ${age} tuổi \nTôi có hoài bão để theo đuổi";
```